daemon/execdriver/native/create.go

```diff
@@ -219,8 +219,12 @@ func (d *driver) setupMounts(container *configs.Config, c *execdriver.Command) e
 
 	// Filter out mounts that are overriden by user supplied mounts
 	var defaultMounts []*configs.Mount
+	_, mountDev := userMounts["/dev"]
 	for _, m := range container.Mounts {
 		if _, ok := userMounts[m.Destination]; !ok {
+			if mountDev && (m.Destination == "/dev" || strings.HasPrefix(m.Destination, "/dev/")) {
```

Comment:
I think it can be just `mountDev && strings.HasPrefix(m.Destination, "/dev/")`, because here `mountDev && m.Destination == "/dev"` is always false, because of `!ok`.
So if `mountDev == false` then we just skip this condition, if `mountDev == true`, then `m.Destination == "/dev" is always false.